06_building_stuff/src/components/Main.jsx

```diff
@@ -65,10 +71,10 @@ export default function Main() {
         <button
           className="filter-btn"
           onClick={() => {
-            setRestaurantData(
-              restaurantData.filter((item) => item.info.avgRating >= 4)
+            setFilteredList(
```

Comment:
using `setFilteredList` instead of `setRestaurantData` method because I dont want to mutate the original data so I am using a filteredList for rendering data on the page.